wiki/confidence-level-modified-event-aggregation.md

```diff
@@ -1,7 +1,8 @@
 # ConfidenceLevelModified event Aggregation
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?